app/models/publications/provider_recruitment_performance_report.rb

```diff
@@ -2,5 +2,9 @@ module Publications
   class ProviderRecruitmentPerformanceReport < ApplicationRecord
     belongs_to :provider
     validates :cycle_week, :publication_date, presence: true
+
+    def reporting_date
```

Comment:
I think the name of this one is confusing. I would think 'reporting date' would be the date the report was created, which is the publications date. Maybe `reporting_end_date`. `report_range_end_date`